src/grafana_dashboards/hydra.json.tmpl

```diff
@@ -0,0 +1,693 @@
+{
```

Comment:
so there are a chunk of variables that we are not using, mainly the ones `juju` related, i'd say to scrap them as we dont benefit from them (right now at least)